source/unified-test-format/tests/valid-pass/initialCollectionData-collectionOptions.yml

```diff
@@ -0,0 +1,28 @@
+description: initialCollectionData-collectionOptions.yml
+
+schemaVersion: '1.5'
+
+createEntities:
+  - client:
+      id: &client0 client0
+  - database:
+      id: &database0 database0
+      client: *client0
+      databaseName: &database0Name database0
+  - collection:
+      id: &collection0 collection0
+      database: *database0
+      collectionName: &collection0Name coll0
+
+initialData:
+  - collectionName: *collection0Name
+    databaseName: *database0Name
+    collectionOptions:
+      capped: true
+      size: 500
```

Comment:
Would it make sense for this test to do something more to assert that `collectionOptions` were actually applied when creating the collection? As-is, I'd worry this could pass even if a driver ignored `collectionOptions`.

I realize we can't observe APM events for this, but perhaps you could use [`$collStats`](https://docs.mongodb.com/manual/reference/operator/aggregation/collStats/#storagestats-document) aggregation stage (with a 3.4+ server version requirement) or the [`collStats`](https://docs.mongodb.com/manual/reference/command/collStats/) command and match the result with `capped` and `size`.